components/Scroller/Scroller.module.scss

```diff
@@ -37,11 +37,11 @@
   }
 }
 
-.hideControls {
-  div>button:first-child {
-    display: none;
-  }
-  div>button:last-child {
-    display: none;
-  }
-}
\ No newline at end of file
+// .hideControls {
```

Comment:
If this isn't relative any more please delete this, as opposed to commenting out.